### DISCUSSION TITLE
SystemVerilog Assertions Basics

### DISCUSSION THREAD

**Discussion by subbdue** [subbdue/systemverilog.io#5] [Website Comments]
Please use this Discussions thread for questions and comments on the article `SystemVerilog Assertions Basics`

**Comment by skull-squadron** [comment]
Writing nit:

- https://www.systemverilog.io/verification/sva-basics

`s/english/English/g`

- https://www.systemverilog.io/verification/blueprint-for-formal-verification/ (correct form here)

Even in down style, capitalization is occasionally essential for disambiguation of meaning in such cases like "english" is spin on a ball while "English" is either a language or a people from a subset of the UK. Similarly, "earth" is soil while "Earth" is the planet. Down style, taken too far, obscures meaning rather than appearing anti-18th century script capitalizing every other word. That's all I have. Thanks for making useful content.

**Comment by dhanunay** [comment]
In cover property why we don't use implication operators  ?
